coalib/results/Diff.py

```diff
@@ -352,6 +352,7 @@ def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
```

Comment:
Also should we display proper error message in `delete_lines` after catching the error from `_get_changes`.